test/service/job/test_job_config.py

```diff
@@ -1,17 +1,16 @@
 import unittest
-import os
 
 from celery.canvas import Signature
 
 from service.job.job_config import JobConfig, ConfigParseException,\
-    UnknownJobTypeException, RequestParameterException
+    UnknownJobTypeException
 
 
 class JobConfigTest(unittest.TestCase):
 
     def test_valid(self):
         job_config = JobConfig("test/resources/configs/config_valid")
-        job1 = job_config.generate_job("job1", "test_user").chain
+        job1 = job_config.generate_job("job1", "test_user", {}).chain
```

Comment:
vielleicht doch in den parametern default '= {}' parameter einbauen dann brauch mann es hier nicht jedes mal angeben

Reply:
Siehe: https://stackoverflow.com/questions/26320899/why-is-the-empty-dictionary-a-dangerous-default-value-in-python

Das hatte ichn erst so, aber habe es korrigiert nachdem einer der Code-Checker gemeckert hat. https://github.com/dainst/cilantro/pull/98/commits/4f8b06a9262ad6bbbde7ec5299c61437cf2fbd23

Reply:
verstehe

Reply:
s.o.